packages/wonder-blocks-form/src/components/__docs__/checkbox-group.stories.js

```diff
@@ -0,0 +1,262 @@
+import {StyleSheet} from "aphrodite";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+import {Choice, CheckboxGroup} from "@khanacademy/wonder-blocks-form";
+import {LabelLarge, LabelXSmall} from "@khanacademy/wonder-blocks-typography";
+
+import type {StoryComponentType} from "@storybook/react";
+
+import ComponentInfo from "../../../../../.storybook/components/component-info.js";
+import {name, version} from "../../../package.json";
+
+export default {
+    title: "Form / CheckboxGroup",
+    component: CheckboxGroup,
+    parameters: {
+        componentSubtitle: ((
+            <ComponentInfo name={name} version={version} />
+        ): any),
+    },
+};
+
+export const Default: StoryComponentType = (args) => {
+    return (
+        <CheckboxGroup {...args}>
+            <Choice label="Pepperoni" value="pepperoni" />
+            <Choice
+                label="Sausage"
+                value="sausage"
+                description="Imported from Italy"
+            />
+            <Choice label="Extra cheese" value="cheese" />
+            <Choice label="Green pepper" value="pepper" />
+            <Choice label="Mushroom" value="mushroom" />
+        </CheckboxGroup>
+    );
+};
+
+Default.args = {
+    // Required
+    groupName: "toppings",
+    selectedValues: ["pepperoni", "sausage"],
+    onChange: () => {},
+    // Optional
+    label: "Pizza toppings",
+    description: "Choose as many toppings as you would like.",
+};
+
+export const Basic: StoryComponentType = () => {
+    const [selectedValues, setSelectedValues] = React.useState([]);
+
+    return (
+        <CheckboxGroup
+            groupName="toppings"
+            onChange={setSelectedValues}
+            selectedValues={selectedValues}
+        >
+            <Choice label="Pepperoni" value="pepperoni" />
+            <Choice
+                label="Sausage"
+                value="sausage"
+                description="Imported from Italy"
+            />
+            <Choice label="Extra cheese" value="cheese" />
+            <Choice label="Green pepper" value="pepper" />
+            <Choice label="Mushroom" value="mushroom" />
+        </CheckboxGroup>
+    );
+};
+
+Basic.parameters = {
+    docs: {
+        storyDescription: `This is a basic example of a checkbox group.
+        The Wonder Blocks \`CheckboxGroup\` component takes \`Choice\`
+        components as children. One of the \`Choice\` components here
+        includes a description.`,
+    },
+};
+
+export const Error: StoryComponentType = () => {
+    const toppingsError = "You have selected too many toppings";
+    const [selectedValues, setSelectedValues] = React.useState([
+        "pepperoni",
+        "sausage",
+        "cheese",
+        "pepper",
+    ]);
+    const [error, setError] = React.useState(toppingsError);
+
+    // Returns an error message if more than 3 items are selected,
+    // and it returns null otherwise.
```

Comment:
**nitpick (non-blocking):** This return `undefined` and it's totally fine b/c if that's the case, then we won't be passing down this `error` prop and we'll be preventing flow errors. It would be nice to change this comment to mention the `undefined` part.